integration-cli/docker_cli_commit_test.go

```diff
@@ -253,6 +253,7 @@ func TestCommitChange(t *testing.T) {
 		"--change", "EXPOSE 8080",
 		"--change", "ENV DEBUG true",
 		"--change", "ENV test 1",
+		"--change", "ENV PATH /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
```

Comment:
can we use a different PATH here to be sure that it overrides the default PATH ? ( i.e., just `/usr/sbin:/usr/bin:/sbin:/bin` )